plugins/outputs/prometheus_client/prometheus_client_tls_test.go

```diff
@@ -1,114 +0,0 @@
-package prometheus_client_test
```

Comment:
Was this intentional?

Reply:
Yes, these tests do not work with Go 1.13 and essentially test that the Go TLS library works.